### DISCUSSION TITLE
How do I become a contributor?

### DISCUSSION THREAD

**Discussion by cdt-eth** [nounsDAO/nouns-monorepo#397] [Ideas/Contribute]
How do I become a contributor?

**Comment by solimander** [comment]
Clone [the monorepo](https://github.com/nounsDAO/nouns-monorepo), run though the quickstart steps, and start poking around at the codebase to see if it lines up with what you like working with. Some of the tools in the stack are: React, TypeScript, Netlify, Bootstrap, ethers.js. That'll probably give you an idea of where might be more comfortable - or uncomfortable. Then we can work through some ideas. We're always working on on-chain image storage, letting people generate Nouns in the playground, experimenting with identity, making more information about Nouns' activity visible (about to push an update for this one), and making the Nouns Protocol more reusable.

**Reply by DanielDanielsson** [reply]
Hi @solimander! 

I've done that part, poked around and I really enjoyed it! I would like to contribute to the project and tried to submit a pull request with a minor documentation fix in the webapp readme for starters. I was not able to push my branch since I don't have write access, is there a way for me to get that? 

Thanks in advance!

**Reply by solimander** [reply]
Hey @DanielDanielsson, happy to hear you're interested in contributing!

You can fork the monorepo and [create a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).

**Reply by brandonleboeuf** [reply]
Hey @solimander. I made a pr from my fork at https://github.com/nounsDAO/nouns-monorepo/pull/659 to address a small bug with the close button in the "View all bids" modal on iOS. 

I don't have the ability to request reviewers, so not sure if you were notified of this PR or not
